lib/init.ts

```diff
@@ -23,22 +23,7 @@ export interface InitParams {
  * Binds credentials and settings to class
  * @param options: initParams
  */
-export function init(options: InitParams) {
-  if (!options) {
-    throw new Error(
-      "Init function should be called with an object argument containing your apiKey and appId"
```

Comment:
these errors should likely be returned right? or do we want people to send init without credentials still (as a way to set the other parameters)? Maybe they can be only dropped if `partial` is true

Reply:
I think it's fine to drop those here as a similar error will be thrown with `sendEvent` when there are no credentials.